javalin/src/test/java/io/javalin/staticfiles/TestStaticFiles.kt

```diff
@@ -258,6 +259,22 @@ class TestStaticFiles {
         }
     }
 
+    @Test
+    fun `only handle resources matching the filter`() {
+        TestUtil.test(Javalin.create { servlet ->
+            // use custom filter - only urls ending with ".css" are handled
+            servlet.addStaticFiles {
+                it.hostedPath = "/"
+                it.directory = "/public"
+                it.location = Location.CLASSPATH
+                it.filter = Predicate { path -> path.endsWith(".css") }
+            }
+        }) { _, http ->
+            assertThat(http.get("/styles.css").status).isEqualTo(200) // access to other locations in /public is not allowed
```

Comment:
I don't understand this comment: `// access to other locations in /public is not allowed`. Copy past error?

Reply:
yes, copy paste, fixed